range_sort.c

```diff
@@ -0,0 +1,47 @@
+#include "range.h"
+
+void bubbleSortAscending(const float* inputArray, int numOfReadings, float* sortedArray)

```

Comment:
Look for builtin functions in preference to building your own sorting. See http://www.cplusplus.com/reference/cstdlib/qsort/
Otherwise you will need to test sorting :)

Reply:
I wasn't aware of built in sort functions, took this code from online , ensured the sorting of this function in console by printing sorted array, will write test case for the same